src/Quasar/Error.purs

```diff
@@ -18,20 +18,30 @@ module Quasar.Error where
 
 import Prelude
 
+import Data.Either (Either)
 import Control.Monad.Eff.Exception (Error, error, message)
 
 data QError
   = NotFound
+  | Forbidden
   | Error Error
 
 instance showQError ∷ Show QError where
   show NotFound = "NotFound"
+  show Forbidden = "Forbidden"
   show (Error err) = "(Error " <> show err <> ")"
 
 printQError ∷ QError → String
 printQError NotFound = "Resource not found"
+printQError Forbidden = "Resource is unavailable, please, authorize"
```

Comment:
Perhaps "Resource is unavailable, authorization is required first" or something like that? If "please authorize" is presented to a user at some point they might think it is something they can do, whereas the likelihood is it's something we've done wrong.